cmd/main.go

```diff
@@ -302,18 +306,35 @@ func main() {
 		crdUtils = controllers.NewDummyCRDUtils()
 	}
 
+	gc := garbagecollector.NewGarbageCollectors(db, log.WithField("pkg", "garbage_collector"), hostApi, clusterApi, objectHandler, lead)
+
+	if Options.EnableDeregisterInactiveGC {
+
+		deregisterWorker := thread.New(
+			log.WithField("garbagecollector", "Deregister Worker"),
+			"Deregister Worker",
+			Options.DeregisterWorkerInterval,
+			gc.DeregisterInactiveClusters)
+
+		deregisterWorker.Start()
+		defer deregisterWorker.Stop()
+	}
+
+	if Options.EnableDeregisterInactiveGC {
```

Comment:
should be EnableDeletedUnregisteredGC

Reply:
What is the difference between `Deregister` and `Delete` ? (soft and hard deletes?)

Reply:
Maybe we can have a single env variable of string type string? can get the values:
* off
* deregister
* delete
* all

for example.

Reply:
It's not deleting it's deregistering.  
there is a function that PermanentClustersDeletion

Reply:
deregister -> remove from the user point of view
delete -> full elimination from the DB

deregister will happen after 30 days since the last update while the deletion happens 15 days since the cluster was deregisterd